external/fv3fit/fv3fit/_shared/config.py

```diff
@@ -153,6 +153,9 @@ class DenseHyperparameters:
         spectral_normalization: whether to apply spectral normalization to hidden layers
         save_model_checkpoints: if True, save one model per epoch when
             dumping, under a 'model_checkpoints' subdirectory
+        rectify_outputs: if True, add a ReLu activation layer as the last layer after
```

Comment:
Can you rename this to `nonnegative_outputs`? It's a little easier for me to wrap my head around (and probably for someone new). Honestly I have no idea why it's called a rectified linear unit.
```suggestion
        rectify_outputs: if True, add a ReLU activation layer as the last layer after
```